app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,22 @@
+class CommentsController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.create(text: comment_parameters[:text], author_id: current_user.id, post_id: @post.id)
```

Comment:
- [ ] I think a comment is created on behalf of the currently logged in user (current_user) of the app. How about making sure the comments controller should have new and create action.

Reply:
I fixed this.